site2/website/versioned_docs/version-2.6.3/administration-zk-bk.md

```diff
@@ -156,11 +170,12 @@ The [`conf/zookeeper.conf`](reference-configuration.md#zookeeper) file handles t
 
 The [`conf/global-zookeeper.conf`](reference-configuration.md#configuration-store) file handles the configuration for configuration store. The table below shows the available parameters:
 
+
 ## BookKeeper
 
 BookKeeper stores all durable message in Pulsar. BookKeeper is a distributed [write-ahead log](https://en.wikipedia.org/wiki/Write-ahead_logging) WAL system that guarantees read consistency of independent message logs calls ledgers. Individual BookKeeper servers are also called *bookies*.
 
-> To manage message persistence, retention, and expiry in Pulsar, refer to [cookbook](cookbooks-retention-expiry.md).
+> To manage message persistence, retention, and expiry in Pulsar, refer to [cookbook](cookbooks-retention-expiry).
```

Comment:
is it because the file does not have a .md postfix or the markdown doesn't need the .md postfix